### ISSUE TITLE
Fething series cloudflare error 502

### ISSUE THREAD

**Issue by Smallkan** [simplymemes/crunchyroll-dl#6]
Fetching series:

![image](https://user-images.githubusercontent.com/9128062/53511904-3501ce80-3aa0-11e9-94f5-d0aa1eb176ca.png)

By episode is normal, but not catching but the series / seasons :(

**Comment by simplymemes** [comment]
Crunchyroll was probably down at the time you performed this command, does it still not work for you?

**Comment by Noctunus** [comment]
Same problem here - crunchy is up and running, single episode fetch is working but with full series I always get:

`
i crunchyroll-dl v1.1.3
i Successfully initiated USA Crunchyroll session
i Attempting to login...
i Successfully logged in!
i Logged in with a premium account.
? Current session expires on Wed Apr 03 2019 [...]
i Attempting to fetch series...
? Attempting to fetch https://www.crunchyroll.com/kanokon/
! Error fetching series: Request failed with status code 503
! Crunchyroll appears to be down.
i Logging out...
i Logging out...
`

I also fiddled a bit around with the code and added a custom user agent just like anime-dl uses it but unfortunately that didn't work either.

Also double checked: 
 * (de)activate unblocking
 * changes in request url: http/https, left out the "www." added "/" at the end
 * checked several different series which were available right before the url changes

**Comment by simplymemes** [comment]
@Noctunus Thanks for the testing! I have singled this problem out to CR making Cloudflare have their protection page for the series page (which is the only page I load not using the API). I have added a Cloudflare bypass for this, and haven't had any problems yet...

Also, performing your test case, it came to my attention that I also completely forgot to test mature content, and have fixed that as well (before mature content would not work).

**Comment by Noctunus** [comment]
@simplymemes Retested and it's working like a charm. Thanks for the quick response, fix and also adding the mature content :). 

btw: I've some ideas for improvements and I've messed around a bit with the older version to test some stuff. Can I reach you somehow to directly discuss it or shall I open feature requests for it?

**Comment by simplymemes** [comment]
@Noctunus Feel free to create a new issue for that.
